src/test/debuginfo/basic-types.rs

```diff
@@ -44,6 +43,8 @@
 // gdb-check:$13 = 2.5
 // gdb-command:print f64
 // gdb-check:$14 = 3.5
+// gdb-command:print s
+// gdb-check:$15 = "Hello, World!"
```

Comment:
If you turn this into a `gdbr-check` then the check will only be done if the GDB version available has native support for Rust (and thus should print the string slice as expected). Relatedly, a `gdbg-` prefix will cause the line to only be executed if the GDB version does not support Rust. Lines with a `gdb-` prefix are executed in both cases.